alma/lib/Model/CartData.php

```diff
@@ -61,6 +62,17 @@ public static function cartInfo($cart)
         ];
     }
 
+    /**
+     * Item of Current Cart
+     *
+     * @param Cart $cart
+     * @return array
+     */
+    public static function cartItems($cart)
```

Comment:
why don't you use directly `getCartItems` ?
I don't see what the `cartItems` method add to the `getCartItems` method ... (you should remove this method and use `getCartItems` instead

Reply:
Because before getCartItems was private fonction but I probably edit that by necessary and forget to remove them. Thanks